docs/docs/writing-a-long-lived-client.md

```diff
@@ -2,34 +2,29 @@
 
 ## Prerequisites
 
-This tutorial assumes you have read and understood [Writing your first enclave](writing-hello-world.md).
+Complete the following tutorials.
 
-## Introduction
+* [Running your first enclave](running-hello-world.md).
+* [Writing your first enclave](writing-hello-world.md).
+* [Writing your own enclave host](writing-your-own-enclave-host.md).
 
-The introductory hello-world sample (see [Running your first enclave](running-hello-world.md)
-and [Writing your first enclave](writing-hello-world.md)) implements a scenario where each run is independent, so the
-client does not need to maintain any state between runs: it can generate a new one-time key each time and there is no
-need to track a multi-step protocol.
+## Introduction
 
-Most real-world scenarios are more complex than this. In particular, the interaction between a client and
-enclave may unfold over an extended period of time, meaning the client needs to be able to resume successfully after a
-restart.
+In the [hello-world sample](running-hello-world.md) and [writing your first enclave](writing-hello-world.md) tutorial,
```

Comment:
`tutorials` I guess

Reply:
Rewrote the sentence as:

Each run is independent in the [hello-world tutorial](https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/db439412-8f74-4776-ad3d-65777df6960c/repository/download/ConclaveCore_SdkDocs/4539:id/docs.zip!/running-hello-world.html) and the [writing your first enclave](https://r3-conclave.artifacts.teamcity.com/presignedTokenAuth/db439412-8f74-4776-ad3d-65777df6960c/repository/download/ConclaveCore_SdkDocs/4539:id/docs.zip!/writing-hello-world.html) tutorial. In such cases, the client doesn't need to maintain any state between runs.